README.md

```diff
@@ -38,7 +38,7 @@ ShopifyAPI::Session.temp(domain: domain, token: token, api_version: api_version)
 end
 ```
 
-The `api_version` attribute can take the string or symbol name of any known version and correctly coerce it to a `ShopifyAPI::ApiVersion`.  You can find the currently defined versions [here](https://github.com/Shopify/shopify_api/blob/master/lib/shopify_api/api_version.rb#L58), follow these [instructions](#adding-aditional-api-versions) to add additional version definitions if needed.
+The `api_version` attribute can take the string or symbol name of any known version and correctly coerce it to a `ShopifyAPI::ApiVersion`.  You can find the currently defined versions [here](https://github.com/Shopify/shopify_api/blob/master/lib/shopify_api/defined_versions.rb), follow these [instructions](#adding-aditional-api-versions) to add additional version definitions if needed.
```

Comment:
Should the defined versions link instead point to your commit like this? https://github.com/Shopify/shopify_api/blob/b9f431a9f593c3ef77c16f490d1399a48923526e/lib/shopify_api/defined_versions.rb

re: #570

Reply:
I don't think we have to, by moving it to a file that only contains the version definitions we don't need to constantly update the link to a specific commit as the versions change.